microsoft/PowerToys
# [KBM] Use icon in the KeyboardManagerEditor

## Summary of the Pull Request

**What is this about:**

Moved icon from the `KeyboardManager` to the resources of the `KeyboardManagerEditor` project.

**What is include in the PR:** 

**How does someone test / validate:** 

Build projects in the `keyboardmanager` folder, verify no build errors.
Verify icon is in place when opening the editor.

## Quality Checklist


## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
